_nodes/0011.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Discrete right fibrations
+macrolib: topos
+---
+
+A displayed category $E$ over $B$ is a *discrete right fibration* when for each
+morphism $f:x\to y$ and displayed object $\bar{y}\in E_y$, there exists a unique pair
+of $\bar{x}\in E_x$ and $\bar{f}:\bar{x}\to\Sub{f}\bar{y}$.
```

Comment:
This definition is a little annoying because it speaks of the strict uniqueness of an object. This reminds me that the invariant notion of discrete fibration is actually presheaf, LOL. Anyway, we should think a bit about whether there is a better and more invariant definition that we can use.

For instance, a "right fibration" can be defined to be a cartesian fibration for which all vertical maps are isomorphisms, or equivalently, all maps are cartesian. This corresponds to a fibration whose fibers are all groupoids. A (strictly) discrete right fibration is one whose fibers are all *strictly* discrete groupoids, but it seems that in between these there would be a reasonable notion for fibrations whose fibers are groupoids that are essentially discrete, i.e. they are "h-sets". The thing to be careful about here is, I suppose, that one the result would be that the category of discrete fibrations in this sense is not equivalent to presheaves, but actually *bi-equivalent*. But that also feels like the right statement.

What do you think?